src/js/components/StatusIcon.js

```diff
@@ -0,0 +1,56 @@
+import React, { PropTypes, Component } from "react";
+
+import Icon from "#SRC/js/components/Icon";
+
+const ICONS_STATE = {
+  RUNNING: {
+    id: "circle-check",
+    color: "green"
+  },
+  TRANSITION: {
+    id: "spinner",
+    color: "grey"
+  },
+  STOPPED: {
+    id: "circle-minus",
+    color: "light-grey"
+  },
+  WARNING: {
+    id: "yield",
+    color: "yellow"
+  }
+};
+
+/**
+ * Status Icon is meant to be used across
+ * the application where it need to display any of the state below
+ * All green (RUNNING), transition (DEPLOYING, RECOVERING), Warning
+ *
+ * @class StatusIcon
+ * @extends {Component}
+ */
+class StatusIcon extends Component {
```

Comment:
The purpose of this component is to isolate the state/style and logic shaping the way we use only what is necessary, no more no less. That way keeping the style and logic consistent in all cases.

That being said you did bring a great point and suggestion to use a constant where I can accomplish the same goal. thanks.

Now, I do not want this to be specific to service status because these icons states will be used in other places.
The naming although is similar to serviceStatus I found that naming the state following a act rather than the color is a better approach.

I would change for something else but that was the best option at the time.